DT2UA6090JXP,"Traceback (most recent call last):
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 475, in <module>
    restockItem(hostnames[i])
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 418, in restockItem
    singleStage(""rhs_restock"", ""Restock"", item)
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 335, in singleStage
    driver.find_element_by_xpath(""//div[7]/div/div/table/tbody/tr/td[2]/span/strong/a"").click()
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 394, in find_element_by_xpath
    return self.find_element(by=By.XPATH, value=xpath)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 978, in find_element
    'value': value})['value']
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 321, in execute
    self.error_handler.check_response(response)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py"", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {""method"":""xpath"",""selector"":""//div[7]/div/div/table/tbody/tr/td[2]/span/strong/a""}
  (Session info: chrome=76.0.3809.100)

"
DT2UA6120XQG,"Traceback (most recent call last):
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 475, in <module>
    restockItem(hostnames[i])
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 418, in restockItem
    singleStage(""rhs_restock"", ""Restock"", item)
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 340, in singleStage
    clickTableItem(""sc_req_item.sc_task.request_item_table"", conditions, ""Number"")  # Select the Task automatically
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 304, in clickTableItem
    table = tableToArray(table_id)
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 252, in tableToArray
    table = driver.find_element_by_id(table_id)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 978, in find_element
    'value': value})['value']
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 321, in execute
    self.error_handler.check_response(response)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py"", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {""method"":""css selector"",""selector"":""[id=""sc_req_item.sc_task.request_item_table""]""}
  (Session info: chrome=76.0.3809.100)

"
DT2UA6060KM4,"Traceback (most recent call last):
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 475, in <module>
    restockItem(hostnames[i])
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 418, in restockItem
    singleStage(""rhs_restock"", ""Restock"", item)
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 340, in singleStage
    clickTableItem(""sc_req_item.sc_task.request_item_table"", conditions, ""Number"")  # Select the Task automatically
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 304, in clickTableItem
    table = tableToArray(table_id)
  File ""C:/Users/mrh19a/Documents/Scripts/PyAutomation/main.py"", line 252, in tableToArray
    table = driver.find_element_by_id(table_id)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 360, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 978, in find_element
    'value': value})['value']
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\webdriver.py"", line 321, in execute
    self.error_handler.check_response(response)
  File ""C:\Users\w10box15\AppData\Local\Programs\Python\Python37-32\lib\site-packages\selenium\webdriver\remote\errorhandler.py"", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {""method"":""css selector"",""selector"":""[id=""sc_req_item.sc_task.request_item_table""]""}
  (Session info: chrome=76.0.3809.100)

"
